graylog2-server/src/main/java/org/graylog2/indexer/datanode/RemoteReindexingMigrationAdapter.java

```diff
@@ -20,14 +20,18 @@
 import org.graylog2.indexer.migration.IndexerConnectionCheckResult;
 import org.graylog2.indexer.migration.RemoteReindexMigration;
 
+import java.net.MalformedURLException;
```

Comment:
this is a new import but I don't see the line of code that needs it?

Reply:
Slipped back when merging master. I'll remove it again.